From 3acbfbcbe708a17fd0b6c20858205a6073595909 Mon Sep 17 00:00:00 2001
From: Jeremy Fitzhardinge <jeremy.fitzhardinge@citrix.com>
Date: Thu, 2 Sep 2010 21:29:34 -0700
Subject: [PATCH 07/26] xen/evtchn: rename retrigger_dynirq -> irq

It is just the generic mechanism for retriggering an event.

Signed-off-by: Jeremy Fitzhardinge <jeremy.fitzhardinge@citrix.com>
(cherry picked from commit c6a16a778f86699b339585ba5b9197035d77c40f)

Signed-off-by: Ian Campbell <ijc@hellion.org.uk>
---
 drivers/xen/events.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/xen/events.c b/drivers/xen/events.c
index c0c69ff..34e18a1 100644
--- a/drivers/xen/events.c
+++ b/drivers/xen/events.c
@@ -1204,7 +1204,7 @@ static void ack_dynirq(unsigned int irq)
 		unmask_evtchn(evtchn);
 }
 
-static int retrigger_dynirq(unsigned int irq)
+static int retrigger_irq(unsigned int irq)
 {
 	int evtchn = evtchn_from_irq(irq);
 	struct shared_info *sh = HYPERVISOR_shared_info;
@@ -1372,7 +1372,7 @@ static struct irq_chip xen_dynamic_chip __read_mostly = {
 
 	.eoi		= ack_dynirq,
 	.set_affinity	= set_affinity_irq,
-	.retrigger	= retrigger_dynirq,
+	.retrigger	= retrigger_irq,
 };
 
 static struct irq_chip xen_percpu_chip __read_mostly = {
@@ -1402,7 +1402,7 @@ static struct irq_chip xen_pirq_chip __read_mostly = {
 
 	.set_affinity	= set_affinity_irq,
 
-	.retrigger	= retrigger_dynirq,
+	.retrigger	= retrigger_irq,
 };
 
 int xen_set_callback_via(uint64_t via)
-- 
1.7.5.4

